index.html

```diff
@@ -17,11 +17,53 @@ <h2>My favorite activities:</h2>
         <li>Stargazing</li>
         <li>Running in the woods</li>
         </ul>
-    <h3>Books I like:</h3>
+    <h2>Books I like:</h2>
         <ol>
-        <li>A Year of Magical Thinking</li>
-        <li>All the Light We Cannot See</li>
-        <li>Everything I Never Told You</li>
+        <li>A Year of Magical Thinking by Joan Dideon</li>
+        <li>All the Light We Cannot See by Anthony Duerr</li>
+        <li>Everything I Never Told You by Celest Eng</li>
         </ol>
+
+    <h3>Favorite Foods:</h3>
+        <ol>
```

Comment:
This ol should also be indented the same as the h3 above (the li's stay where they are)